src/main/java/appeng/client/gui/implementations/GuiMEMonitorable.java

```diff
@@ -210,7 +213,7 @@ else if( btn == this.searchBoxSettings )
 		}
 	}
 
-	public void re_init()
+	private void re_init()
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00100)